### ISSUE TITLE
ID and Class Additions to Sections.

### ISSUE THREAD

**Issue by JacobHorne** [ThemeFuse/Unyson#3483]
I need the ability to add an #ID or Classname to sections in the page builder. I need to add ids and classes to images/elements as well. Please advise.

**Comment by danyj** [comment]
Browse the git please,simple google search for "add id to section unyson" returns

https://github.com/ThemeFuse/Unyson/issues/275

**Comment by JacobHorne** [comment]
I read the entire thread. Lots of stuff going on in there from 2015. I don't really see a solution here.

Why would this not be an option in a theme?

**Comment by danyj** [comment]
If your theme did not add possibility for it than you must code yourself. Don't know what you are using
but the post I referenced is bit complicated for non techs and you must combine dynamic CSS and the ID. I also do not know if your theme will let you extend the actual framework.  


I did something like this for our Creatus theme

in shortcode options.php add 

```
			'id' => array(
				'type' => 'unique',
			),
```

in shortcode view.php on top

```
$id 			= fw_akg('id',$atts);
$id_out			=  'prefix-'.$id;
```
in shortcode view.php on element wrap 

`<div id="<?php echo esc_attr($id_out) ?>"`

testdrive the section or any element options here https://labs.themezly.net/ for more ideas
